docs/src/physics/coordinate_systems.md

```diff
@@ -0,0 +1,62 @@
+# Coordinate systems
+
+Every grid in Oceananigans is associated with an _extrinsic_ coordinate system.
+The extrinsic coordinate system is either _Cartesian_ or _geographic_
+(spherical / latitude-longitude).
+
+## Cartesian coordinates
+
+Cartesian coordinates are used by [`RectilinearGrid`](@ref) and provide a local, flat representation
+of space with coordinates ``\boldsymbol{x} = (x, y, z)`` and unit vectors
+``\boldsymbol{\hat x}``, ``\boldsymbol{\hat y}``, and ``\boldsymbol{\hat z}``.
+By convention, ``\boldsymbol{\hat x}`` points east, ``\boldsymbol{\hat y}`` points north,
+and ``\boldsymbol{\hat z}`` points "upward", opposite to the direction of gravitational acceleration.
+
+We denote time with ``t``, partial derivatives with respect to time ``t`` or a coordinate ``x``
+with ``\partial_t`` or ``\partial_x``, and the gradient operator
+```math
+\boldsymbol{\nabla} \equiv \partial_x \boldsymbol{\hat x} + \partial_y \boldsymbol{\hat y} + \partial_z \boldsymbol{\hat z} \, .
+```
+Horizontal gradients are
+```math
+\boldsymbol{\nabla}_h \equiv \partial_x \boldsymbol{\hat x} + \partial_y \boldsymbol{\hat y} \, .
+```
+
+We use ``u``, ``v``, and ``w`` to denote the velocity components in the ``x``, ``y``, and ``z`` directions,
+such that the three-dimensional velocity is
+``\boldsymbol{v} = u \boldsymbol{\hat x} + v \boldsymbol{\hat y} + w \boldsymbol{\hat z}``.
+We use ``\boldsymbol{u} = u \boldsymbol{\hat x} + v \boldsymbol{\hat y}`` to denote the horizontal velocity.
+
+## Geographic coordinates
+
+Geographic (or spherical) coordinates ``(λ, φ, z)`` represent longitude, latitude, and
+the vertical coordinate and are used by [`LatitudeLongitudeGrid`](@ref) and [`OrthogonalSphericalShellGrid`](@ref).
+The corresponding unit vectors are ``\boldsymbol{\hat λ}`` (eastward), ``\boldsymbol{\hat φ}`` (northward),
```

Comment:
Should we clarify "local unit vectors" to convey the fact that in contrast with RectilinearGrid, these unit vectors change with grid location?

Reply:
In the extrinsic coordinate system it is grid independent right?

Reply:
Feel free to tweak/edit